File: src/test/java/com/larryhsiao/clotho/color/BlueIntTest.java
package com.larryhsiao.clotho.color;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

/**
 * Test for {@link BlueInt}
 */
public class BlueIntTest{
    /**
     * Normal case
     */
    @Test
    void normalCase() {
        assertEquals(
            160,
            new BlueInt(0xFFffA0).value()
        );
    }
}

File: src/test/java/com/larryhsiao/clotho/color/BlueIntTest.kt
package com.larryhsiao.clotho.color

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

/**
 * Test for [BlueInt]
 */
class BlueIntTest {
    @Test
    fun simple() {
        assertEquals(
            160,
            BlueInt(0xFFffA0).value()
        )
    }
}